### ISSUE TITLE
Cannot change my Google id if I want to change my identity

### ISSUE THREAD

**Issue by alankent** [tcmg/adventure-issues#75]
**Is your feature request related to a problem? Please describe.**
At the "sign in with Google" page I misclicked and picked the wrong user id. I have tried to reload the page, but the sign in with Google link remembers my past choice and I am not given an opportunity to change my user id.

**Describe the Idea or Solution you'd like**
There should be some way to force selection of user id. E.g. make the "connect with Google" button always ask me which user id to use, not remember the last one. That would let me log in with a different user id by starting again. Or at the end of the login flow where it shows the badge, give me a way to "start again".

**Why?**
A misclick locks you into the wrong user id. As an expert I know I can create an incognito window, but if any attendee uses the wrong id at login they may get stuck. (Incognito windows also force you to enter passwords etc to login.)

**Comment by tcmg** [comment]
Good point Alan, we're tracking this and we'll add a logout and also destroy account feature.
